recipes/qt/all/test_package/test_package.pro

```diff
@@ -0,0 +1,7 @@
+SOURCES += test_package.cpp
+
+HEADERS += greeter.h
+
+QT -= GUI
```

Comment:
This produces an error when building the Qt package with `options.GUI=False`:

    Project ERROR: Unknown module(s) in QT: gui